src/lib/_imports/trumps/s-hero-banner.css

```diff
@@ -0,0 +1,188 @@
+
+body > main {
+  margin-inline: unset;
+  padding-block: unset;
+  width: unset;
+}
+
+main {
+  padding-block: unset;
+  width: unset;
+}
```

Comment:
> [!WARNING]
> These very broad styles would apply to every page on every CMS. They may not have ill effect yet, but they are extraneous and potentially conflicting.

Perhaps these are the styles you mean to have in `s-hero-banner/s-hero-banner.demo.css`.